postreise/analyze/generation/summarize.py

```diff
@@ -181,8 +181,8 @@ def get_storage_time_series(scenario, area, area_type=None, storage_e=False):
     :param bool storage_e: if set, return time series energy of storage devices
         instead of power generation. Default to False.
     :return: (*pandas.Series*) -- time series of total storage power generation,
-        if storage_e is not set, otherwise, time series of total storage energy,
-        index: time stamps, column: storage power/storage energy
+        if ``storage_e`` is not set, otherwise, time series of total storage energy,
+        index: timestamps, column: storage power/storage energy
```

Comment:
We don't really have a `column` here, do we? I think something like `values: storage power/storage energy` might be more intuitive.

Reply:
Done